include/xtensor-sparse/xsparse_tensor_traits.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef XSPARSE_XSPARSE_TENSOR_TRAITS_HPP
+#define XSPARSE_XSPARSE_TENSOR_TRAITS_HPP
+
+// #include <xtensor-sparse/xsparse_tensor_traits.hpp>
+
+#include "xsparse_expression.hpp"
+#include "xsparse_array.hpp"
+#include "xsparse_tensor.hpp"
+
```

Comment:
`xxx_traits` headers are usually utility files included in a lot of other files. I think it would be better to use forward declarations of `xsparse_array` and `xsparse_tensor` rather than including the headers defining them.